module-domain/src/main/kotlin/site/yourevents/invitation/service/InvitationService.kt

```diff
@@ -55,4 +56,12 @@ class InvitationService(
 
         return invitation.qrUrl
     }
+
+    override fun deleteInvitation(invitationId: UUID, deleted: Boolean) {
+        val invitation = findById(invitationId)
+            ?: throw InvitationNotFoundException()
+
+        invitation.updateDeleted()
+        invitationPersistencePort.delete(invitation)
```

Comment:
소프트 delete 방식으로 초대장 삭제 기능을 구현하는 만큼, `delete()` 메서드는 읽는 사람으로 하여금 오해를 불러 일으킬 수 있다고 생각합니다.. 😭 

```suggestion
        invitationPersistencePort.save(invitation)
```

다음과 같이 메서드명을 변경하면 64번째 line의 로직을 통해 보다 직관적으로 이해할 수 있을 것 같다는 생각입니다!

Reply:
아 그렇군요!! 사실 soft-delete는 엄연히 deleteMapping으로 진행되는게 아니라서 흠.... 진짜 삭제는 아닌데 delete가 맞나 싶었는데 좋은 의견 감사합니다!! 수정하였습니다.